Google.Api.Gax.Grpc.Tests/Rest/RestMethodTest.cs

```diff
@@ -35,9 +35,7 @@ public void CreateRequest_WithRequestNumericEnumJson(bool value, string expected
     public void CreateRequest_WithHttpOverrides()
     {
         var rule = new HttpRule { Get = "/v2/def/{name}" };
-        var methodDescriptor = TestServiceReflection.Descriptor.Services
-            .Single(svc => svc.Name == "Sample")
-            .FindMethodByName("SimpleMethod");
+        var methodDescriptor = GetMethod("Sample", "SimpleMethod");
```

Comment:
Is it worth it to check that the override overrides even when there were no rules? Just so that we don't change it back to what it was?

Reply:
In terms of adding a test? Yes, I can do that.